src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -183,14 +185,27 @@ def execute(self):
 
         # fetch the protectedLFNs list
         try:
+            self.protectedLFNs = set()
```

Comment:
I am not sure I understand why we involve the garbage collector here, this line is only to assure we start the cycle with a fresh zeroed `self.protectedLFNs` set. Which if we fail to update on the very next line then we just interrupt the  cycle two lines bellow. It will introduce no new objects in memory, because `self.protectedLFNs` is an attribute to the object.  

The way we were running right now was by relying that in case anything bad happens with the call to the central services,  `WMStatserver` would do one of the followin two things - either raise an error or return an empty list. I have put that line there to be safe than sorry, because an eventual bug introduced in WMStatServer, may lead to change of behavior on its side and then we could be in a big trouble here.